packages-native/qr-code/package.json

```diff
@@ -1,28 +1,32 @@
 {
-    "name": "qr-code",
-    "widgetName": "QRCode",
-    "version": "0.0.1",
-    "license": "Apache-2.0",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/mendix/widgets-resources.git"
-    },
-    "config": {
-        "widgetPath": "../test-project/mxproject/widgets",
-        "projectPath": "../test-project/mxproject"
-    },
-    "packagePath": "com.mendix.widget.native",
-    "scripts": {
-        "start": "..\"/../node_modules/.bin/pluggable-widgets-tools\" start:ts:native -- --subprojectPath packages-native/qr-code",
-        "build": "..\"/../node_modules/.bin/pluggable-widgets-tools\" build:ts:native --subprojectPath packages-native/qr-code",
-        "release": "..\"/../node_modules/.bin/pluggable-widgets-tools\" release:ts:native --subprojectPath packages-native/qr-code",
-        "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
-        "test": "..\"/../node_modules/.bin/jest\" --config ../../scripts/test/jest.native.config.js",
-        "version": "npm run release",
-        "postversion": "..\"/../node_modules/.bin/ts-node\" ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
-    },
-    "dependencies": {
-        "react-native-qrcode-svg": "^5.3.1",
-        "@native-mobile-resources/util-widgets": "^1.0.0"
-    }
+  "name": "qr-code",
+  "widgetName": "QRCode",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {
+    "widgetPath": "../test-project/mxproject/widgets",
+    "projectPath": "../test-project/mxproject"
+  },
+  "packagePath": "com.mendix.widget.native",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:ts:native -- --subprojectPath packages-native/qr-code",
+    "build": "pluggable-widgets-tools build:ts:native --subprojectPath packages-native/qr-code",
+    "release": "pluggable-widgets-tools release:ts:native --subprojectPath packages-native/qr-code",
+    "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test": "jest --config ../../scripts/test/jest.native.config.js",
+    "version": "npm run release",
+    "postversion": "ts-node ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+  },
+  "dependencies": {
+    "react-native-qrcode-svg": "^5.3.1",
+    "@native-mobile-resources/util-widgets": "^1.0.0"
+  },
+  "devDependencies": {
+    "@mendix/pluggable-widgets-tools": "8.9.0",
+    "react-native-svg": "^9.6.4"
```

Comment:
Shouldnt SVG also be dependency instead?

Reply:
👍